src/EA.Weee.RequestHandlers/Mappings/AatfContactDataMap.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace EA.Weee.RequestHandlers.Mappings
+{
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Core.AatfReturn;
+    using EA.Weee.DataAccess;
+    using EA.Weee.Domain;
+    using EA.Weee.Domain.AatfReturn;
+    using System.Linq;
+
+    public class AatfContactDataMap : IMap<AatfContactData, AatfContact>
+    {
+        public AatfContact Map(AatfContactData source)
+        {
+            if (source != null)
+            {
+                Country country = new Country(source.AddressData.CountryId, source.AddressData.CountryName);
```

Comment:
Here pass the country in via the AatfContactData source, we can't do a new.

Reply:
I've realized that I don't need a country object to create an AATF, so just using the country Id that is already passed in.